config/clusters/utoronto/staging.values.yaml

```diff
@@ -76,7 +76,7 @@ jupyterhub:
             subPath: "{username}"
     image:
       name: quay.io/2i2c/utoronto-image
-      tag: f3069be5964c
+      tag: 18b55a7bcb96
```

Comment:
Warning that without quotes, this could cause problem every 0.35% of the times when the hex string is entirely numerical.